easy-ui-react/src/Badge/Badge.module.scss

```diff
@@ -0,0 +1,95 @@
+@use "../styles/common" as *;
+
+.root {
+  @include font-style("small-button");
+  border-radius: design-token("shape.border_radius.lg");
+  display: inline-flex;
+  // Ensures the background colors do not bleed out of the border radius edges
+  overflow: hidden;
+}
+
+.primary,
+.secondary {
+  display: inline-flex;
+  padding: design-token("space.0.5");
+}
+
+.primary {
+  background: component-token("badge", "primary.color.background");
+  color: component-token("badge", "primary.color.text");
+}
+
+.secondary {
+  background: component-token("badge", "secondary.color.background");
+  color: component-token("badge", "secondary.color.text");
+}
+
+// Wrapper element around the text to normalize the hor and ver padding
+.text {
+  display: inline-flex;
+  padding: design-token("space.0.25") design-token("space.0.5");
+}
+
+// prettier-ignore
```

Comment:
reads a little better in IDEs as single lines. although admittedly when wrapping is enabled (like here) it reads less friendly